cmd/server/cmd/root.go

```diff
@@ -16,7 +16,7 @@ import (
 var language string
 
 func init() {
-	RootCmd.PersistentFlags().StringVarP(&language, "language", "l", "zh", "Set the language (default is 'zh')")
+	RootCmd.PersistentFlags().StringVarP(&language, "language", "l", "en", "Set the language")
 }
 
 var RootCmd = &cobra.Command{
```

Comment:
I don't have the current version of the program to analyze, but generally speaking, this code snippet appears to be missing a section where it handles errors gracefully if an invalid input is provided for "language". This error handling might not cause problems currently, but without proper checks and logging mechanisms, it could later become problematic when encountering rare input formats or incorrect values such as non-string data types that cannot be passed through `var` declaration.

For optimization, you can consider adding appropriate validations at command-line parsing stage or before storing into global state variables like `RootCmd`, avoiding any runtime panic. For example:

```go
  
const (
   DEFAULT_LANGUAGE = "zh"
)

func init() {
	var language string

	switch flag := RootCmd.PersistentFlags().GetString("language"); flag {
	case "":
		language = DEFAULT_LANGUAGE
	default:
	.language = flag
	}

	if !utils.IsValidLanguage(language) {
	  // Handle invalid language value
	}
```
This change would help ensure that the user's choice of language will always be validated against known valid languages, even in the case where no value has been explicitly specified. Additionally, more granular validation logic (such as checking for specific codes associated with supported locales) should also be implemented here depending upon what type of system or application this script operates on.